examples/sim_policy.py

```diff
@@ -53,7 +53,6 @@ def query_yes_no(question, default='yes'):
                         type=int,
                         default=1000,
                         help='Max length of episode')
-    parser.add_argument('--speedup', type=float, default=1, help='Speedup')
```

Comment:
why are you removing this?

Reply:
It wasnt being used in `rollout` so it had no effect.